src/box/memtx_bitset.c

```diff
@@ -218,6 +218,14 @@ bitset_index_iterator_next(struct iterator *iterator, struct tuple **ret)
 	return 0;
 }
 
+static int
+bitset_index_iterator_next(struct iterator *iterator, struct tuple **ret)
+{
+	if (bitset_index_iterator_next_raw(iterator, ret) != 0)
+		return -1;
+	return memtx_prepare_result_tuple(ret);
+}
+
```

Comment:
Here is a problem: we should call `memtx_prepare_result_tuple` all time in `get` and `next` functions because tuple in space can be compressed and decompressed at the same time. We can implement `prepare_result_tuple` function pointer in `engine_vtab` or `index_vtab` but it is additional call for spaces without compression